docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -697,6 +701,40 @@ by specifying `quarkus.rest-client.multipart-post-encoder-mode` in your
 clients created with the `@RegisterRestClient` annotation.
 All the available modes are described in the link:https://netty.io/4.1/api/io/netty/handler/codec/http/multipart/HttpPostRequestEncoder.EncoderMode.html[Netty documentation]
 
+=== Receiving Multipart Messages
+REST Client Reactive also supports receiving multipart messages.
+As with sending, to parse a multipart response, you need to create a class that describes the response data, e.g.
+
+[source,java]
+----
+class FormDto {
+    @FormParam("file")
+    @PartType(MediaType.APPLICATION_OCTET_STREAM)
+    public File file;
+
+    @FormParam("otherField")
+    @PartType(MediaType.TEXT_PLAIN)
+    public String textProperty;
+}
+----
+
+Then, create an interface method that corresponds to the call and make it return the `FormDto`:
+[source,java]
+----
+    @GET
+    @Produces(MediaType.MULTIPART_FORM_DATA)
+    @Path("/get-file")
+    FormDto data sendMultipart();
+----
+
+At the moment, multipart response support is subject to the following limitations:
+
+- files sent in multipart responses can only be parsed to `File`, `Path` and `FileDownload`
+- each field of the response type has to be annotated with `@PartType` - fields without this annotation are ignored
+
+REST Client Reactive needs to prepare to handle multipart return types. If you have an interface method that produces `multipart/form-data`, the return type will be picked up automatically. However, if you intend to use the `ClientBuilder` API to parse a response as multipart, you need to annotate your DTO class with `@MultipartForm`.
```

Comment:
What does `REST Client Reactive needs to prepare to handle multipart return types` mean? I don't get it

Reply:
I meant that the quarkus extension needs to know the classes used for multipart responses upfront, I'll try to rephrase it

Reply:
is the current version better?

Reply:
Much better, thanks